src/components/Articles.jsx

```diff
@@ -0,0 +1,39 @@
+ import {useParams} from 'react-router-dom'; 
+import { useState, useEffect } from 'react';
+import {getArticles } from '../utils/api';
+import axios from 'axios';
+import '../App.css';
+
+
+const Articles = () => { 
+  const [articles, setArticles] = useState([]); 
+  
+    useEffect(() => {
+  axios.get(`https://yoris-nc-news.onrender.com/api/articles`)
+      .then(({data}) => {       
+         return setArticles(data);
+      });
+    },[])
+  
+  return (
+  <section>
+    <h2>LIST OF ARTICLES</h2>
+    <ul className="">
+      {articles.map((article) => {
+return (
+  <li className = "articlecount" key={article.article_id}>
```

Comment:
To get rid of unwanted bullet points, you can set the `list-style-type` css property of `<li>` elements  to `none` in App.css